### DISCUSSION TITLE
useMemoizedFn 的 ts 能优化下吗?

### DISCUSSION THREAD

**Discussion by vaynevayne** [alibaba/hooks#2326] [Q&A] [answered]
<img width="905" alt="image" src="https://github.com/alibaba/hooks/assets/124666577/16b42d60-3393-4698-bdf1-91cf34172f35">

传一个不存在的 a, 都不会校验的

**Comment by liuyib** [comment, marked as answer]
![image](https://github.com/alibaba/hooks/assets/38221479/ea9bb43d-8cac-4abb-ad62-1b89443141b1)

这里 any，你把你截图这个函数类型完善下就好了

**Reply by liuyib** [reply]
我验证过了，不是 ahooks useMemoizedFn 的问题。这个讨论关了~
